uirules/uirules.go

```diff
@@ -56,3 +57,14 @@ var FormatJSONText = tailsql.UIRewriteRule{
 		return s
 	},
 }
+
+// LinkURLText is a UI rewrite rule to add links for URL-shaped results.
+var LinkURLText = tailsql.UIRewriteRule{
+	Value: regexp.MustCompile(`^https?://\S+$`),
+	Apply: func(col, s string, _ []string) any {
+		if _, err := url.Parse(s); err == nil {
+			return template.HTML(fmt.Sprintf(`<a href="%s" referrerpolicy=no-referrer rel=noopener>`, s))
```

Comment:
I'm concerned that `%s` might include a `"><script>...` from some user data. Am I misunderstanding `template.HTML`?

Reply:
It could in general, but in this case we are only plugging it in if a URL parser accepts it as valid, which should rule out that sort of shenanigan.

Reply:
I made it render the URLs string explicitly, so we can be sure we are getting proper escapes.